model/trace/rpc.yaml

```diff
@@ -94,117 +53,40 @@ groups:
-            - id: already_exists
-              brief: ALREADY_EXISTS
-              value: 6
-            - id: permission_denied
-              brief: PERMISSION_DENIED
-              value: 7
-            - id: resource_exhausted
-              brief: RESOURCE_EXHAUSTED
-              value: 8
-            - id: failed_precondition
-              brief: FAILED_PRECONDITION
-              value: 9
-            - id: aborted
-              brief: ABORTED
-              value: 10
-            - id: out_of_range
-              brief: OUT_OF_RANGE
-              value: 11
-            - id: unimplemented
-              brief: UNIMPLEMENTED
-              value: 12
-            - id: internal
-              brief: INTERNAL
-              value: 13
-            - id: unavailable
-              brief: UNAVAILABLE
-              value: 14
-            - id: data_loss
-              brief: DATA_LOSS
-              value: 15
-            - id: unauthenticated
-              brief: UNAUTHENTICATED
-              value: 16
+      - ref: rpc.grpc.status_code
+        tag: tech-specific
         requirement_level: required
-        brief: "The [numeric status code](https://github.com/grpc/grpc/blob/v1.33.2/doc/statuscodes.md) of the gRPC request."
-      - id: request.metadata
-        type: template[string[]]
+      - ref: rpc.grpc.request.metadata
+        tag: tech-specific
         requirement_level: opt_in
-        brief: >
-          gRPC request metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values.
-        note: >
-          Instrumentations SHOULD require an explicit configuration of which metadata values are to be captured.
-          Including all request metadata values can be a security risk - explicit configuration helps avoid leaking sensitive information.
-        examples: ['rpc.grpc.request.metadata.my_custom_metadata_attribute=["1.2.3.4", "1.2.3.5"]']
-      - id: response.metadata
-        type: template[string[]]
+      - ref: rpc.grpc.response.metadata
+        tag: tech-specific
         requirement_level: opt_in
-        brief: >
-          gRPC response metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values.
-        note: >
-          Instrumentations SHOULD require an explicit configuration of which metadata values are to be captured.
-          Including all response metadata values can be a security risk - explicit configuration helps avoid leaking sensitive information.
-        examples: ['rpc.grpc.response.metadata.my_custom_metadata_attribute=["attribute_value"]']
 
   - id: rpc.jsonrpc
     prefix: rpc.jsonrpc
     type: span
     extends: rpc
     brief: 'Tech-specific attributes for [JSON RPC](https://www.jsonrpc.org/).'
     attributes:
-      - id: version
-        type: string
+      - ref: rpc.jsonrpc.version
+        tag: tech-specific
```

Comment:
Isn't this tag too broad? What if someone else uses also `tech-specific` elsewhere? Maybe `grpc-tech-specific` or smt? Are there so many attributes in `gprc` that inheriting is better than simply picking the ones we really want to list here? What is the benefit of inheriting and not listing all of them? 🤔

Reply:
Yep, agreed!